src/test/java/base/BaseTest.java

```diff
@@ -110,9 +112,9 @@ public AlertsFramesWindowsPage getAlertsFramesWindowsPage() {
         return new AlertsFramesWindowsPage(getDriver());
     }
 
-    public BookStoreApplicationPage getBookStoreApplicationPage() {
+    public BookStorePage getBookStorePage() {
```

Comment:
After clicking the BookStoreApplicationMenu we are moving to BookStorePage (URL https://demoqa.com/books, sub-menu Book Store). So BookStoreApplicationPage is the same as BookStorePage.